code/src/components/NewTodo.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState, useMemo } from 'react';
+import todos from 'reducers/todos';
+import { useDispatch } from 'react-redux';
+import DatePicker from 'react-datepicker';
```

Comment:
Impressive how much you have stretched your react muscles 💪 and used these extra libraries/functionalities, I have so many to look up now 😃